airbyte-integrations/connectors/destination-bigquery/src/test-integration/java/io/airbyte/integrations/destination/bigquery/typing_deduping/BigQuerySqlGeneratorIntegrationTest.java

```diff
@@ -851,6 +856,48 @@ public void softReset() throws InterruptedException {
     assertEquals(0, rawUntypedRows);
   }
 
+  @Test
+  public void testMigrateFromV1toV2() throws InterruptedException {
```

Comment:
for when we're resolving merge conflicts with https://github.com/airbytehq/airbyte/pull/28890 - I wonder if it's best to just leave this method in each destination's test implementation and having an abstract `testMigrateFromV1ToV2` method in the base class? This is the only test that needs to prepopulate a v1 table...